kernel_hardening_checker/checks.py

```diff
@@ -673,6 +673,19 @@ def add_cmdline_checks(l: List[ChecklistObjType], arch: str) -> None:
     if arch in ('X86_64', 'X86_32'):
         l += [CmdlineCheck('self_protection', 'clipos', 'iommu', 'force')]
 
+    # 'self_protection', 'a13xp0p0v'
+    l += [OR(CmdlineCheck('self_protection', 'a13xp0p0v', 'lockdown', 'confidentiality'),
+             AND(KconfigCheck('self_protection', 'kspp', 'LOCK_DOWN_KERNEL_FORCE_CONFIDENTIALITY', 'y'),
+                 CmdlineCheck('self_protection', 'a13xp0p0v', 'lockdown', 'is not set')))]
+    l += [OR(CmdlineCheck('self_protectioыn', 'a13xp0p0v', 'module.sig_enforce', '1'),
+             KconfigCheck('cut_attack_surface', 'kspp', 'MODULES', 'is not set'),
+             AND(KconfigCheck('self_protection', 'kspp', 'MODULE_SIG_FORCE', 'y'),
+                 CmdlineCheck('self_protection', 'a13xp0p0v', 'module.sig_enforce', 'is not set')))]
```

Comment:
Please change all your new checks according to the fix https://github.com/a13xp0p0v/kernel-hardening-checker/commit/8256e96d6e31786c035dd82a7a7438bc3476e031.